package.json

```diff
@@ -291,12 +291,12 @@
                 "command": "clangd.switchheadersource",
                 "key": "Alt+o",
                 "mac": "Alt+cmd+o",
-                "when": "editorTextFocus"
+                "when": "(resourceLangId == c || resourceLangId == cpp || resourceLangId == cuda-cpp || resourceLangId == objective-c || resourceLangId == objective-cpp) && editorTextFocus"
             },
             {
                 "command": "clangd.typeHierarchy",
                 "key": "Shift+Alt+t",
-                "when": "editorTextFocus"
+                "when": "(resourceLangId == c || resourceLangId == cpp || resourceLangId == cuda-cpp || resourceLangId == objective-c || resourceLangId == objective-cpp) && editorTextFocus"
```

Comment:
nit: `resourceLangId == c` can be omitted because there is no inheritance and thus no type hierarchies in C

(clangd doesn't currently implement type hierarcy for Objective-C either, but the concept of inheritance does exist in Objective-C and a future version of the server might support type hierarchy for it, so `resourceLangId == objective-c` can be kept)

Reply:
Good point, I have updated the branch.